### PR TITLE
Mdct 2302 okta OIDC v2

### PR DESCRIPTION
**Pull request by tblackwe**
### Description
Adding support for OIDC SSO, controllable by SSM params

### Related ticket

MDCT-2302

### How to test
Can only be tested on main
### Author checklist

- [X] I have performed a self-review of my code
- [] ~I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary~
- [X] I have updated the documentation, if necessary

### CODE REVIEW

**Review by BearHanded — DISMISSED** [review]
Do we need any placeholder stuff in the UI, or is it all handled between cognito and okta handshaking? Are all the ssm params in place in main?

File: services/ui-auth/serverless.yml [hunk 1]
```diff
@@ -129,6 +148,7 @@ resources:
           - email
           - openid
           - profile
+          - phone
```
Comment (BearHanded) [inline services/ui-auth/serverless.yml:151]: is this going to affect existing users in any way, or just pull it through?

Comment (tblackwe) [reply services/ui-auth/serverless.yml:151]: phone slipped through my copy paste. removing it now. ty.

No placeholder stuff in UI, gets handled from cognito/handshake.

intentionally leaving SSM out in 1st iteration to ensure getting same SAML behavior. Will add SSM param in after initial deploy and validate OIDC.

Comment (tblackwe) [reply services/ui-auth/serverless.yml:151]: the SSM for SAML exists, and 2 of the 3 for OIDC 
image

**Review by karla-vm — APPROVED** [review]

**Review by BearHanded — APPROVED** [review]

**Pull request merged**